index.html

```diff
@@ -120,9 +120,9 @@ <h3>Best Design</h3>
                    <p>Partner</p>
                 </div>
                 <div class="flex img_footer">
-                    <img src="/media/hamza.jpeg" width="150px" alt="">
-                    <img src="/media/google.jpg" width="150px" alt="">
-                    <img src="/media/github.jpg" width="150px" alt="">
+                    <img src="/media/hamza.jpeg" width="150" alt="">
+                    <img src="/media/google.jpg" width="150" alt="">
+                    <img src="/media/github.jpg" width="150" alt="">
```

Comment:
- [ ] As stated above,  Please follow the design provided to implement some features, Compare the two images again:

--------------------------------------Original-----------------------------------------------
![nav6](https://user-images.githubusercontent.com/55635977/128220633-dc126f06-f7ee-44e8-9ab1-67cd824ecb3c.png)

--------------------------------------Yours-----------------------------------------------
![nav5](https://user-images.githubusercontent.com/55635977/128220656-85cc4cb6-2f0a-4341-ad54-adcc96b82be6.png)

Reply:
This issue is not fully resolved 😿

Reply:
Is this necesary when I already get a second review and he approved those parts? can you please work with the second reviewer cause I don't have the time to play with reviewers. Why the second one approve these parts and you the tthird one come and say that they are not good

Reply:
Hi, kindly note that the second reviewer simply highlighted on how you can improve a particular section. We're all working together ensure that you come out with a more solid project. Cheers! 🥰